### PR TITLE
Update gems

### PR DESCRIPTION
**Pull request by eugenk**
Just a regular update.

Note that the secure_headers update introduced deprecation warnings. I already migrated it to be compatible to the upcoming major version.

### CODE REVIEW

File: config/initializers/security_headers.rb [hunk 1]
```diff
@@ -1,13 +1,13 @@
-::SecureHeaders::Configuration.configure do |config|
-  config.hsts                   = {:max_age => 99, :include_subdomains => true}
+::SecureHeaders::Configuration.default do |config|
+  config.hsts                   = 'max_age=99; include_subdomains=true'
   config.x_frame_options        = 'DENY'
   config.x_content_type_options = "nosniff"
-  config.x_xss_protection       = {:value => 1, :mode => false}
+  config.x_xss_protection       = 'value=1; mode=false'
```
Comment (eugenk) [inline config/initializers/security_headers.rb]: No thanks. You should stop suggesting this as soon as #1570 gets merged.

File: config/initializers/security_headers.rb [hunk 2]
```diff
@@ -1,13 +1,13 @@
-::SecureHeaders::Configuration.configure do |config|
-  config.hsts                   = {:max_age => 99, :include_subdomains => true}
+::SecureHeaders::Configuration.default do |config|
+  config.hsts                   = 'max_age=99; include_subdomains=true'
```
Comment (eugenk) [inline config/initializers/security_headers.rb]: No thanks. You should stop suggesting this as soon as #1570 gets merged.

**Comment by eugenk** [comment]
I removed the capybara update. Let's wait for 2.6.1.

**Comment by ebolloff** [comment]
:+1:

**Pull request merged**